packages/components/src/form/FormLabel.tsx

```diff
@@ -0,0 +1,38 @@
+import { cns } from '@stcland/utils'
+
+//*****************************************************************************
+// Interface
+//*****************************************************************************
+interface Props {
+  labelText?: string; // If labelText, will still render errorText
+  errorText?: string; // appended to label text in red if profvided
+  className?: string; // className string applied to root of this component
+  required?: boolean; // required fields
+  requiredClass?: string; // class for required fields
+  htmlFor?: string;
+}
+
+//*****************************************************************************
+// Components
+//*****************************************************************************
+const FormLabel = function ({
+  labelText,
+  className,
+  required,
+  requiredClass,
```

Comment:
should we change this prop to be `customStyles`? and also let users update either `root` or `required`?